internal/app/kdk/init.go

```diff
@@ -119,29 +127,29 @@ func InitKdkConfig(logger logrus.Entry) error {
 
 	// Define Additional volume bindings
 	for {
-		prompt := promptui.Prompt {
-			Label: "Would you like to mount additional docker host directories into the KDK? [y/n]",
-			Default: "",
+		prompt := promptui.Prompt{
+			Label:     "Would you like to mount additional docker host directories into the KDK? [y/n]",
```

Comment:
Did you run this through an autoformatter?  Can you put it in the Makefile?

Reply:
`gofmt -s -w cmd internal main.go`